src/styles/product_card.scss

```diff
@@ -0,0 +1,93 @@
+@import 'button';
+@import 'stars';
+
+$bodyfont: Roboto, sans-serif;
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+  padding: 0;
+  font-family: $bodyfont;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+
+  &:hover {
+    transition: 0.3s;
+    transform: scale(1.2);
```

Comment:
The transform: scale(1.2) on hover might cause layout issues by affecting the surrounding elements. Consider using a different approach or ensuring that the surrounding layout can accommodate this scaling without causing overflow or layout shifts.